.github/workflows/performance.yml

```
@@ -0,0 +1,160 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - '6.[1-9]'
```

Actually, I think the example I gave in the previous discussion was incorrect. If we are not backporting this to <= 6.1 then we should make this exclude 6.1 as well.
```suggestion
      - '6.[2-9]'
```